compiler/tests-common/org/jetbrains/kotlin/test/KotlinTestUtils.java

```diff
@@ -430,7 +431,18 @@ public static String doLoadFile(String myFullDataPath, String name) throws IOExc
     }
 
     public static String doLoadFile(@NotNull File file) throws IOException {
-        return FileUtil.loadFile(file, CharsetToolkit.UTF8, true);
+        try {
+            return FileUtil.loadFile(file, CharsetToolkit.UTF8, true);
+        }
+        catch (FileNotFoundException fileNotFoundException) {
+            // Display the absolute path of the file that wasn't loaded
```

Comment:
You're explaining the code behavior in a comment.
The code you wrote is clear enough in that regard, no need for the comment :)

Reply:
This sound better?
```
/*
 * Unfortunately, the FileNotFoundException will only show the relative path in it's exception message.
 * This clarifies the exception by showing the full path.
 */
```

Reply:
A lot better, you're explaining the reason for the change, not the code itself! :)